document-store/src/main/java/org/hypertrace/core/documentstore/expression/operators/AggregationOperator.java

```diff
@@ -4,6 +4,7 @@ public enum AggregationOperator {
   AVG,
   COUNT,
   DISTINCT, // This operator generates an array of distinct values
+  DISTINCT_ARRAY, // This operator generates an array of distinct values
```

Comment:
The comment in the existing code (`This operator generates an array of distinct values`) is a bit misleading.
DISTINCT can be used with normal fields or array fields (after unwinding them).